static/js/github-create-branch-options.js

```diff
@@ -45,11 +45,9 @@ $(document).ready(() => {
 });
 
 const createUrlForGH = (uuid, multiGHInstance) => {
-	const jiraHost = $("#jiraHost").val();
-	const jwt = $("#jwt").val();
   return uuid ?
-    `github/${uuid}/create-branch?issueKey=${issueKey}&issueSummary=${issueSummary}&jiraHost=${jiraHost}&jwt=${jwt}&ghRedirect=to&multiGHInstance=${multiGHInstance}` :
-    `github/create-branch?issueKey=${issueKey}&issueSummary=${issueSummary}&jiraHost=${jiraHost}&jwt=${jwt}&multiGHInstance=${multiGHInstance}`;
+    `github/${uuid}/create-branch?issueKey=${issueKey}&issueSummary=${issueSummary}&ghRedirect=to&multiGHInstance=${multiGHInstance}` :
```

Comment:
Did you tested it that it still works without `jwt=${jwt}`

Reply:
Yep. jiraHost is put to session by jwtSymmetricMiddleware and is used from there afterwards